scripts/which_python.py

```diff
@@ -0,0 +1,2 @@
+import sys
+print("PYTHON_INTERPRETER =", sys.executable)
```

Comment:
This should be moved to a log in `livereduce.py` to be something like (first message is in there already)
```python
logger.info('logging started by user \'' + os.environ['USER'] + '\'')
logger.info(f'using python interpreter {sys.executable}')
```
and this file be removed